src/commands/createFunctionApp/stacks/getStackPicks.ts

```diff
@@ -176,3 +194,10 @@ function removeHiddenStacksAndProperties(stacks: FunctionAppStack[]): void {
         }
     }
 }
+
+export function shouldShowEolWarning(eolDate: string): boolean {
+    const sixMonthsFromNow = new Date();
+    sixMonthsFromNow.setMonth(sixMonthsFromNow.getMonth() + 6);
```

Comment:
If it's possible to do on one line, I say do it. Especially since the variable name makes it clear what is happening here.

```suggestion
    const sixMonthsFromNow = new Date(new Date().setMonth(new Date().getMonth() + 6));
```